### DISCUSSION TITLE
How to Implement "SizeChanged"

### DISCUSSION THREAD

**Discussion by yaoyulian** [AvaloniaUI/Avalonia#8473] [Q&A] [answered]
In WPF: <<UserControl  SizeChanged="UserControl_SizeChanged">>UserControl  SizeChanged="UserControl_SizeChanged"
Avalonia do not have "SizeChanged". I tryed "LayoutUpdated", But it made UI halted. Please Add "SizeChanged".

**Comment by maxkatz6** [comment, marked as answer]
Upd: now we have an easier to use SizeChanged event, thanks to @robloo !

Listen for a PropertyChanged event and check Bounds property.

**Reply by maxkatz6** [reply]
Or you can bind to a Bounds property directly

**Comment by robloo** [comment]
This really should be a feature request and an issue. If there is time I will implement it. It's annoying not to have when using existing code from WPF/UWP/WinUI.

**Reply by timunie** [reply]
@robloo I see you are already working on this. Do you want me to move this discussion into a feature request? You need to close this then in your PR. If you react with 👍 it's enough for me.

**Reply by robloo** [reply]
Yea, you can move it over to an issue. That will help track things for the future when the PR is closed.
